x-pack/plugins/ml/server/models/data_recognizer/modules/apm_transaction/ml/high_mean_transaction_duration.json

```diff
@@ -0,0 +1,35 @@
+{
+  "job_type": "anomaly_detector",
+  "groups": [
+    "apm"
+  ],
+  "description": "Detect transaction duration anomalies across transaction types for your APM services",
+  "analysis_config": {
+    "bucket_span": "15m",
+    "detectors": [
+      {
+        "detector_description": "high_mean('transaction.duration.us') by 'transaction.type' partitionfield='service.name'",
```

Comment:
Is there something we can come up with which reads better than the default description? Something like `High mean transaction duration` or `High duration by transaction type`

![image](https://user-images.githubusercontent.com/7405507/86353056-a1757a80-bc5e-11ea-85bf-1b494881495e.png)

Reply:
updated to `high duration by transaction type for an APM service`